### DISCUSSION TITLE
Placeholder You dont have island

### DISCUSSION THREAD

**Discussion by WolverStones** [BG-Software-LLC/SuperiorSkyblock2#246] [Denied Feature Requests] [answered]
If the player does not have an island, the placeholder will not be empty but there will be text for example (You do not own an island)

**Comment by Tptuaazn** [comment]
%superior_island_exists%

**Comment by OmerBenGera** [comment, marked as answer]
You can use %superior_island_exists% and use javascript to get this working.
